newsfragments/40029.significant.rst

```diff
@@ -1 +1,18 @@
-Removed deprecated ``allow_raw_html_descriptions`` option from UI Trigger forms.
+Removed deprecated airflow configuration ``webserver.allow_raw_html_descriptions`` from UI Trigger forms.
+
+* Types of change
+
+  * [ ] DAG changes
+  * [x] Config changes
+  * [ ] API changes
+  * [ ] CLI changes
+  * [ ] Behaviour changes
+  * [x] Plugin changes
```

Comment:
Actually this was not changing anything on plugins. Just the raw HTML was a hack like plugins to be able to inject JavaScript to the forms... beneficial as well as malicious one.
```suggestion
  * [ ] Plugin changes
```

Reply:
docs(newsfragmenst): update 40029 change type